Gemfile.lock

```diff
@@ -103,7 +103,7 @@ GEM
       xcodeproj (>= 1.13.0, < 2.0.0)
       xcpretty (~> 0.3.0)
       xcpretty-travis-formatter (>= 0.0.3)
-    fastlane-plugin-wpmreleasetoolkit (1.3.1)
+    fastlane-plugin-wpmreleasetoolkit (2.3.0)
```

Comment:
Just to note that I [double-checked our list of Breaking Changes when bumping from `1.x.x` to `2.x.x`](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/CHANGELOG.md#breaking-changes-1) and confirmed that none of those breaking changes are impacting `simplenote-android`, so it's ok not to have any other changes about this major version bump in this PR other than the `ENV['FL_RELEASE_TOOLKIT_DEFAULT_BRANCH'] = 'trunk'`  👍